Python/Tests/TestAdapterTests/TestResultParserTests.cs

```diff
@@ -24,18 +24,36 @@ public class TestResultParserTests {
 
         [TestMethod]
         public void CreatePytestId_FuncInsideClass() {
-            Assert.AreEqual(".\\test2.py::Test_test2::test_A", TestResultParser.CreatePytestId("test2.py", "test2.Test_test2", "test_A"));
+            Assert.AreEqual(
```

Comment:
style: put the `);` on the next line, indented -1
```
hello(
    1,
    2,
    3
);
```

this matches the style used with curly braces:
```
void hello() {
    var a = 1;
}
```

same comment applies below

Reply:
ok